codespell_lib/data/dictionary.txt

```diff
@@ -27128,6 +27409,7 @@ pyramide->pyramid
 pyramides->pyramids
 pyrhon->python
 pyscic->psychic
+pysical->physical
```

Comment:
`pysic`, `pysics`?

Note that there's a [pysic](https://thynnine.github.io/pysic/) Python module...

Reply:
Added. I moved `pysic` to `dictionary_code.txt` for good measure though. I also added `pysically`.